tutorials/intermediate/constrainedlayout_guide.py

```diff
@@ -455,6 +466,21 @@ def docomplicated(suptitle=None):
 ax2 = fig.add_axes(bb_ax2)
 
 ###############################################################################
+# Manually turning off ``constrained_layout``
+# ===========================================
+#
+# ``constrained_layout`` usually adjusts the axes positions on each draw
+# of the figure.  If you want to get the spacing provided by
+# ``constrained_layout`` but then not have it update, then do the initial
```

Comment:
leave out the "then" before "do". This sentence has too much "then" :smile:

Reply:
Thanks - I took out the first "then" so it now reads:

```
# If you want to get the spacing provided by
# ``constrained_layout`` but not have it update, then do the initial
# draw and then call ``fig.set_constrained_layout(False)``.
```